.github/workflows/ci.yml

```diff
@@ -77,7 +81,7 @@ jobs:
           Exit 0
       - name: Push all built packages to MyGet
         # Only push packages on master if they were built (not if testing was skipped)
-        if: steps.test.outcome == 'success' && github.event_name == 'push'
+        if: steps.test.outcome == 'success' && github.event_name == 'push' && matrix.os == 'windows-2019'
```

Comment:
Is there a reason we're only looking for a specific version of Windows OS here?

Reply:
yes, to avoid pushing the packages 3 times

Reply:
should I add a comment to clarify it? Or do you have a better idea how to achieve this in a more beautiful way? 🤔

Reply:
Perhaps just a comment so I don't forget later 😄

Reply:
@Ana06 at'ing you so this doesn't get lost